### PR TITLE
Linux: Fix regression with mask of cursors if platform name is 'xcb'

### PR DESCRIPTION
**Pull request by wwmayer**

### CODE REVIEW

File: src/Gui/View3DInventorViewer.cpp [hunk 1]
```diff
@@ -661,14 +661,12 @@ static QCursor createCursor(QBitmap &bitmap, QBitmap &mask, int hotX, int hotY,
         QImage img = bitmap.toImage();
         img.convertTo(QImage::Format_ARGB32);
         pixmap = QPixmap::fromImage(img);
```
Comment (3x380V) [inline src/Gui/View3DInventorViewer.cpp:663]: I thought it was tested on 'xcb', but apparently not well enough..

Perhaps move definition here, then?
`QPixmap pixmap = QPixmap::fromImage(img);`

Comment (3x380V) [reply src/Gui/View3DInventorViewer.cpp:663]: Ehm, declaration... So the whole patch reads:
```
@@ -656,19 +656,16 @@ static QCursor createCursor(QBitmap &bitmap, QBitmap &mask, int hotX, int hotY,
     Q_UNUSED(dpr)
 #endif
 #ifdef HAS_QTBUG_95434
-    QPixmap pixmap;
     if (qGuiApp->platformName() == QLatin1String("wayland")) {
         QImage img = bitmap.toImage();
         img.convertTo(QImage::Format_ARGB32);
-        pixmap = QPixmap::fromImage(img);
-    } else {
-        pixmap = bitmap;
+        QPixmap pixmap = QPixmap::fromImage(img);
+        pixmap.setMask(mask);
+        return QCursor(pixmap, hotX, hotY);
     }
-    pixmap.setMask(mask);
-    return QCursor(pixmap, hotX, hotY);
-#else
-    return QCursor(bitmap, mask, hotX, hotY);
 #endif
+
+    return QCursor(bitmap, mask, hotX, hotY);
 }
 
 void View3DInventorViewer::createStandardCursors(double dpr)
```
Also thanks for testing that on X and for the fix.

Comment (chennes) [reply src/Gui/View3DInventorViewer.cpp:663]: @3x380V I'm going to merge this as is, but please feel free to submit your suggestion as a follow-up PR.

Comment (3x380V) [reply src/Gui/View3DInventorViewer.cpp:663]: @chennes, that's just a nit, not worth extra time (compiler will deal with it anyway).
Thanks to everyone involved.

**Pull request merged**